src/style.css

```
@@ -1,3 +1,63 @@
+:root {
+  --primary-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 50px;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+.header__logo {
+  height: 40px;
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  position: relative;
+}
+
+.nav__link.is-active {
+  position: relative;
+  color: var(--primary-color);
+}
+
+.nav__item--active .nav__link::after {
+  content: "";
+  position: absolute;
+  bottom: -22.5px;
```

To make tests see this line you should use the correct width for the links and use `0` for the bottom value here
```suggestion
  bottom: 0;
```